code/datums/preferences.dm

```diff
@@ -777,6 +780,19 @@ $(function() {
 			<br><a href="[pref_link]tooltip=2" class="toggle">[crap_checkbox(src.tooltip_option == TOOLTIP_ALT)] Show When ALT is held</a>
 			<br><a href="[pref_link]tooltip=3" class="toggle">[crap_checkbox(src.tooltip_option == TOOLTIP_NEVER)] Never Show</a>
 		</td>
+	</tr>"}
+		LAGCHECK(80)
+		if (rebuild_data["tgui"])
+			rebuild_data["tgui"] = 0
+			data_cache["tgui"] = {"
+	<tr>
+		<th>
+			tgui<span class="info-thing" title="tgui is the UI framework we use for some game windows, and it comes with options!">?</span>
+		</th>
+		<td colspan="2">
+			<a href="[pref_link]tgui_fancy=1" class="toggle">[crap_checkbox(src.tgui_fancy)] ~fancy~ tgui windows</a>
+			<br><a href="[pref_link]tgui_lock=1" class="toggle">[crap_checkbox(src.tgui_lock)] Lock tgui on screen</a>
```

Comment:
They're supposed to 'lock' the windows so they can't go off of the byond screen... I think it's broken in my current implementation for some reason

Reply:
Lock is only for initial window placement, and it constrains window position to the primary monitor (if you have a dual monitor setup).

You should also strongly hint (somehow) that non fancy is slower, because it really is 3-4 times slower to open. On tg this switch toggles between "Fancy (default)" and "Compatible (slower)".